<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTXViewerWidget</class>
 <widget class="QMainWindow" name="PTXViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PTXViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,17,6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>3D Points</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QVTKWidget" name="qvtkWidgetLeft"/>
      </item>
      <item row="1" column="1">
       <widget class="QVTKWidget" name="qvtkWidgetRight"/>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radRGB">
          <property name="text">
           <string>RGB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radDepth">
          <property name="text">
           <string>Depth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radIntensity">
          <property name="text">
           <string>Intensity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radValidity">
          <property name="text">
           <string>Validity</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFlipImage"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExportRGBImage"/>
    <addaction name="actionExportDepthImage"/>
    <addaction name="actionExportValidityImage"/>
    <addaction name="actionExportRGBDImage"/>
    <addaction name="actionExportRGBDVImage"/>
    <addaction name="actionExportIntensityImage"/>
    <addaction name="actionExportPointCloud"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpen_Grayscale_Image">
   <property name="text">
    <string>Open As Grayscale Image</string>
   </property>
  </action>
  <action name="actionOpen_RGBDI_Image">
   <property name="text">
    <string>Open As RGBDI Image</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFlipImage">
   <property name="text">
    <string>Flip Image</string>
   </property>
  </action>
  <action name="actionExportRGBImage">
   <property name="text">
    <string>RGB Image</string>
   </property>
  </action>
  <action name="actionExportDepthImage">
   <property name="text">
    <string>Depth Image</string>
   </property>
  </action>
  <action name="actionExportValidityImage">
   <property name="text">
    <string>Validity Image</string>
   </property>
  </action>
  <action name="actionExportRGBDImage">
   <property name="text">
    <string>RGBD Image</string>
   </property>
  </action>
  <action name="actionExportRGBDVImage">
   <property name="text">
    <string>RGBDV Image</string>
   </property>
  </action>
  <action name="actionExportIntensityImage">
   <property name="text">
    <string>Intensity Image</string>
   </property>
  </action>
  <action name="actionExportPointCloud">
   <property name="text">
    <string>Point Cloud</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
